Formula/xdg-ninja.rb

```diff
@@ -0,0 +1,21 @@
+class XdgNinja < Formula
+  desc "Check your $HOME for unwanted files and directories"
+  homepage "https://github.com/b3nj5m1n/xdg-ninja"
+  url "https://github.com/b3nj5m1n/xdg-ninja/archive/refs/tags/v0.2.0.1.tar.gz"
+  sha256 "f4f9ab4500e7cf865ff8b68c343537e27b9ff1e6068cb1387e516e608f77cec8"
+  license "MIT"
+  head "https://github.com/b3nj5m1n/xdg-ninja.git", branch: "main"
+
+  depends_on "glow"
+  depends_on "jq"
+
+  def install
+    prefix.install "programs/"
+    prefix.install "xdg-ninja.sh" => "xdg-ninja"
+    bin.install_symlink prefix/"xdg-ninja"
+  end
+
+  test do
+    system bin/"xdg-ninja"
```

Comment:
Can we check output here?

Reply:
Kind of. There's no machine readable information in the output, and output isn't guaranteed to stay the constant between updates.

However, there are some sections of the output that will probably stay constant. I could check for those, if you want.

Reply:
Hmmm, no, I guess this is fine.